codecov.yml

```
@@ -5,3 +5,5 @@ coverage:
         target: auto
         threshold: 0.03%
         base: auto
+  comment:
+    after_n_builds: 16 # Prevent early, spurious Codecov reports before all tests finish: https://github.com/kensho-technologies/graphql-compiler/pull/806#issuecomment-730622647. 16 is calculated from the number of jobs to run, which is specified in the .github/workflows/tests_and_lint.yml file: 6 lint jobs (1 job per combination of python-version and lint-flags) and 8 test jobs (1 job per combination of python-version and markers), plus 1 for codecov patch build, plus 1 for the codecov project build.
```

`comment` is actually a top-level section in the `yaml`.

```suggestion
comment:
  after_n_builds: 16 # Prevent early, spurious Codecov reports before all tests finish: https://github.com/kensho-technologies/graphql-compiler/pull/806#issuecomment-730622647. 16 is calculated from the number of jobs to run, which is specified in the .github/workflows/tests_and_lint.yml file: 6 lint jobs (1 job per combination of python-version and lint-flags) and 8 test jobs (1 job per combination of python-version and markers), plus 1 for codecov patch build, plus 1 for the codecov project build.
```